latch_cli/utils.py

```diff
@@ -138,6 +140,63 @@ def hash_directory(dir_path: Path) -> str:
     return m.hexdigest()
 
 
+def generate_temporary_ssh_credentials(ssh_key_path: Path) -> str:
+    # check if there is already a valid key at that path, and if so, use that
+    # otherwise, if its not valid, remove it
+    if ssh_key_path.exists():
+        try:
+            # check if file is valid + print out a fingerprint for the key
+            cmd = ["ssh-keygen", "-l", "-f", ssh_key_path]
+            valid_private_key = subprocess.run(cmd, check=True)
+            cmd = ["ssh-keygen", "-l", "-f", ssh_key_path.with_suffix(".pub")]
+            valid_public_key = subprocess.run(cmd, check=True)
+            if valid_private_key.stdout != valid_public_key.stdout:
+                raise
```

Comment:
`rase` what?

Reply:
this is just a way of getting into the except block below, it doesn't matter what exception im raising